docs/userguides/transactions.md

```diff
@@ -189,7 +202,7 @@ ethereum:
 
 Transaction traces are the steps in the contract the transaction took.
 Traces both power a myriad of features in Ape as well are themselves a tool for developers to use to debug transactions.
-To learn more about traces, see the [traces userguide](./trace.md).
+To learn more about traces, see the [traces userguide](./trace.html).
```

Comment:
(should check this locally) I think `./trace` works as a relative link for both `.md` and `.html`

Reply:
Just checked and `./trace` will need the `.html` to work at the end. `.md` only works when using a text-editor and isn't recognized after the docs are generated.